packages/indexer-common/src/indexer-management/client.ts

```diff
@@ -167,6 +172,22 @@ const SCHEMA_SDL = gql`
     priority: Int
   }
 
+  input ActionOrderByInput {
```

Comment:
Good clear name, but maybe for consistency with other input names remove `Input` from the end?